requirements-dev.txt

```diff
@@ -1,31 +1,2 @@
-# When adding new dependencies here, also add them to the renovate.json to
-# enable auto-merge
-
 -r requirements.txt
-black==19.3b0
-factory-boy==3.0.1
-faker==4.16.0
-flake8==3.8.4
-flake8-bugbear==20.1.4
-flake8-debugger==3.2.1
-flake8-docstrings==1.5.0
-flake8-isort==4.0.0
-flake8-string-format==0.3.0
-flake8-tuple==0.4.1
-gitlint==0.13.1
-isort==4.3.21
-pdbpp==0.10.2
-pre-commit==2.8.2
-pydocstyle==5.1.1
-pytest==6.1.2
-pytest-cov==2.10.1
-pytest-django==4.1.0
-pytest-env==0.6.2
-pytest-factoryboy==2.0.3
-pytest-mock==3.3.1
-pytest-randomly==3.5.0
-pytest-xdist==2.1.0
-python-semantic-release==7.3.0
-requests-mock==1.8.0
-reuse==0.11.1
-syrupy==0.8.4
+-r requirements-devel.txt
```

Comment:
Why this indirection? I find it a bit confusing to have -dev and -devel here

Reply:
The indirection is needed, so that the package build step doesn't install the exact requirements from "requirements.txt" too, that would make the build step useless.
The reasoning behind this is, that the package should install and work with the loosened "install_requires" after all!

I agree the naming is not optimal, actually I would prefer something like "requirements-all.txt" what used to be "requirements-dev.txt" until now (since it loads all the requirements) and "requirements-dev.txt" holding just the dev dependencies.
However that would break our workflow / muscle memory...

Reply:
Hmm, I do like the `-all` suffix, even if it does break muscle memory. Makes it more obvious what's happening, than trying to remember whether to use `-dev` or `-devel` every time.

Reply:
fine with me!